<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,10,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="activityName">
       <property name="toolTip">
        <string>Identifier for the activity configuration</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GraphicsLayoutWidget" name="selectElectrodesPlot"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="currentActivities"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ambient Temperature</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ambient RH %</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Metabolic Activity </string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Clothing Description</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Radiation Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="eventDuration">
       <property name="placeholderText">
        <string>Exposure Duration in minutes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="relativeHumidity">
       <property name="placeholderText">
        <string>Ambient Relative</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ambientTemperature">
       <property name="placeholderText">
        <string>Ambient Temperature</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="metabolicActivity">
       <property name="placeholderText">
        <string>Metabolic Activity Level (Numeric - 0.8 for rest)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="activityDescription">
       <property name="placeholderText">
        <string>Brief Descritption about the activity</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="velocityOfAir">
       <property name="placeholderText">
        <string>Velocity of Ambient Air</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Velocity Of Air m/s</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="clothingFile">
         <property name="placeholderText">
          <string>Location of file describing clothing resistance properties or NUDE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadingClothingFile">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="radiationFluxFile">
         <property name="placeholderText">
          <string>Location of file describing radiation fluxes or NONE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadingRadiationFile">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="loadActivity">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addActivity">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateActivity">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cloneActivity">
       <property name="text">
        <string>Clone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteActivity">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>activityDescription</tabstop>
  <tabstop>eventDuration</tabstop>
  <tabstop>relativeHumidity</tabstop>
  <tabstop>ambientTemperature</tabstop>
  <tabstop>metabolicActivity</tabstop>
  <tabstop>velocityOfAir</tabstop>
  <tabstop>addActivity</tabstop>
  <tabstop>updateActivity</tabstop>
  <tabstop>cloneActivity</tabstop>
  <tabstop>deleteActivity</tabstop>
  <tabstop>save</tabstop>
  <tabstop>selectElectrodesPlot</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
